### PR TITLE
kucoin-profile

### PR DESCRIPTION
**Pull request by kucoinstaking**

### CODE REVIEW

File: validators/terravaloper183kevvgrqvav9sa0lwzhr54jngxw6sac90jvs2/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (emidev98) [inline validators/terravaloper183kevvgrqvav9sa0lwzhr54jngxw6sac90jvs2/README.md:1]: Could you add a header similar to the one from the [example profile](https://github.com/terra-money/validator-profiles/pull/437/files#diff-2db876394628438c041d08254ed57ba608099123cc68a350f10c5a42b54ec1d8R2-R4)

**Review by emidev98 — CHANGES_REQUESTED** [review]
Hey, there are some properties

**Comment by kucoinstaking** [comment]
update kucoin

File: validators/terravaloper183kevvgrqvav9sa0lwzhr54jngxw6sac90jvs2/README.md (hunk 1 above)
Comment (kucoinstaking) [reply validators/terravaloper183kevvgrqvav9sa0lwzhr54jngxw6sac90jvs2/README.md:6]: updated

**Pull request merged**